modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -50,7 +64,7 @@ $ oc patch Network.config.openshift.io cluster \
 +
 [source,terminal]
 ----
-$ watch oc get pod -n openshift-sdn
+$ watch oc get pod -n openshift-multus
```

Comment:
This part still need confirmation from @pliurh ,according to the new procedure doc,after update the network type,"Check there is no pod under namespace ‘openshift-multus’", however the pods multus-XXXX just got recreated and still in running status.  And other pods in namespace openshift-multus like multus-admission-controller-XXXXX and network-metrics-daemon-XXXXX were not recreated.

Reply:
It should be 'wait until all the pods of the DaemonSet `multus` are recreated'.

Reply:
@pliurh @huiran0826 

Is there a specific pattern that a user should look for, like `multus-XXXX`, or is it all pods in the `openshift-multus` namespace?

Reply:
`multus-XXXX`

Reply:
I've updated to include:

> The name of the Multus pods is in form of `multus-<xxxx>` where `<xxxx>` is a random sequence of letters.